aws/resource_aws_lb_target_group.go

```diff
@@ -120,6 +120,17 @@ func resourceAwsLbTargetGroup() *schema.Resource {
 				}, false),
 			},
 
+			"load_balancing_algorithm_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "round_robin",
+				ForceNew: true,
```

Comment:
So I decided to give this branch a try since I really wanted the functionality .. it seems like `ForceNew` shouldn't be required here .. AWS Console seems to be able to do this as just an edit .. TF should too ..

Reply:
I also just changed this to false on my local copy and it seems to work just fine. It was able to apply the new setting properly to the target group.

Reply:
You're right, thanks for the input!